### ISSUE TITLE
Path interpretation issue in read_chroms

### ISSUE THREAD

**Issue by ADablanc** [ethanbass/chromConverter#14]
Hi, 

There is a problem with the read_chrom function, a "/" is added at the beginning and end of path_out which prevents the function from working properly.

if i try thise code (on Windows): 
=========================================================================
library(chromConverter)

if (dir.exists("DATAs/OUTPUT/neg")){
  print("THE DIRECTORY EXIST")
}

dat <- read_chroms(paths = "DATAs/INPUT/neg", format_in = "thermoraw", path_out = "DATAs/OUTPUT/neg")
=========================================================================

 i have this output:
=========================================================================
> library(chromConverter)
> 
> if (dir.exists("DATAs/OUTPUT/neg")){
+   print("THE DIRECTORY EXIST")
+ }
[1] "THE DIRECTORY EXIST"
> 
> dat <- read_chroms(paths = "DATAs/INPUT/neg", format_in = "thermoraw", path_out = "DATAs/OUTPUT/neg")
Error in read_chroms(paths = "DATAs/INPUT/neg", format_in = "thermoraw",  : 
  The export directory '/DATAs/OUTPUT/neg/' does not exist.
=========================================================================

**Comment by ethanbass** [comment]
Hi,
Sorry about this. I'm pretty sure this is fixed in the latest version of chromConverter available from GitHub, which I can see I really need to get uploaded to CRAN. Can you please try downloading from GitHub (e.g. by running `remotes::install_github("ethanbass/chromConverter"`) and try again (after resetting your R session).
Thanks!
Ethan

**Comment by ethanbass** [comment]
Ok. Just to make sure, did you restart your R session after installing it?
If not, the old version is likely still cached.

**Comment by ADablanc** [comment]
Yes, i even restart my computer because i have the issue friday and today.

**Comment by ethanbass** [comment]
I just double checked on a computer in the lab running Windows 10 and I was only able to reproduce this issue using the CRAN version of chromConverter (version 0.2.1) but not the version on GitHub (version 0.3.3), so I'm not sure what's going on.  Can you please run `packageVersion("chromConverter")` to confirm that it says v 0.3.3?

**Comment by ethanbass** [comment]
Maybe you can also confirm that `.Platform$OS.type` returns "windows" on your computer. If not this could explain the problem (though I can't imagine why this would be the case

**Comment by ethanbass** [comment]
Any update on this?

**Comment by ethanbass** [comment]
I'm going to close this due to lack of activity, but feel free to reopen if you're still having this issue and can provide additional details to help me reproduce this issue. Also feel free to try the version on the v0.3.4 branch, which has completely removed the `check_paths` function which is likely underlying this issue. Thanks!
Ethan
